packages/account-v2/src/containers/ProofOfIncome/ProofOfIncomeForm.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { Form, Formik } from 'formik';
+import * as Yup from 'yup';
+import { StandaloneArrowUpFromBracketBoldIcon } from '@deriv/quill-icons';
+import { Button, Divider, Text, useDevice } from '@deriv-com/ui';
+import { FormDropDownField } from '../../components/FormFields';
+import { FormDocumentUploadField } from '../../components/FormFields/FormDocumentUploadField';
+import { FormSubHeader } from '../../components/FormSubHeader';
+import { ProofOfIncomeUtils } from '../../utils/proofOfIncomeUtils';
+
+export const ProofOfIncomeForm = () => {
+    const PoincDocumentsList = ProofOfIncomeUtils();
+    const { isMobile } = useDevice();
+    const listItems = [
+        'The document must be up-to-date and signed by the issuance authority.',
+        'The document must contain a letterhead.',
+        'Invalid or incomplete documents shall be rejected.',
+    ];
+    const documentValidation = Yup.mixed().required('Please upload a file');
+
+    return (
+        <div className='relative z-0 h-full w-full  grid gap-16 '>
+            <Formik>
+                <Form>
+                    <section>
+                        <section>
+                            <div className='mb-[2.4rem]'>
+                                <FormSubHeader>Select Document</FormSubHeader>
+                            </div>
+
+                            <FormDropDownField
+                                className='flex items-start self-stretch h-full border border-[color:var(--border-normal)] border-solid justify-between'
+                                label='Select Document'
+                                list={PoincDocumentsList}
+                                name='Select Document'
+                            />
+                        </section>
+
+                        <section>
+                            <div className='mb-[1.4rem]'>
+                                <FormSubHeader>Document submission</FormSubHeader>
+                            </div>
+
+                            <section className='flex flex-col items-start w-full px-16 py-16 space-y-20 lg:border-solid lg:border-1 border-solid-grey-5  mb-[2.4rem]'>
```

Comment:
Please prefer `md:` over `lg:` so the styles render for all devices other than mobile.